src/Datadog.Trace/Agent/Api.cs

```diff
@@ -204,6 +202,18 @@ private async Task<bool> SendTracesAsync(Span[][] traces, IApiRequest request)
                 // Attempt a retry if the status code is not SUCCESS
                 if (response.StatusCode < 200 || response.StatusCode >= 300)
                 {
+                    if (finalTry)
+                    {
+                        try
+                        {
+                            Log.Error("Failed to submit traces with status code {0} and message: {1}", response.StatusCode, await response.ReadAsStringAsync().ConfigureAwait(false));
```

Comment:
For readability, can we split this into two lines?
```suggestion
                            string responseContent = await response.ReadAsStringAsync().ConfigureAwait(false);
                            Log.Error("Failed to submit traces with status code {0} and message: {1}", response.StatusCode, responseContent );
```